includes/header.inc

```diff
@@ -92,11 +92,16 @@
                             'MacPorts Team' => $trac_url . 'wiki/MacPortsDevelopers',
                             'Becoming a Member' => $guide_url . '#project.membership',
                         );
-                        foreach ($shortcuts as $title => $url) {
-                            if (basename($_SERVER['PHP_SELF']) == $url) {
+						foreach ($shortcuts as $title => $url) {
+							if (basename($_SERVER['PHP_SELF']) == $url) {
```

Comment:
These whitespace changes seem unrelated and unnecessary. The rest of the file uses spaces, not tabs. New code should follow the existing style.

Reply:
@raimue done. Sorry I always forget to change `.vimrc` back to use tabs. Will stick to the new changes.